Makefile

```diff
@@ -1,5 +1,8 @@
 artifact_name       := test-data-generator
-version             := "unversioned"
+version             := unversioned
+docker_registry     := ${AWS_ACCOUNT}.dkr.ecr.${AWS_REGION}.amazonaws.com
```

Comment:
It'll probably be easier just passing in the entire registry url via an env var

Reply:
Actually mate I've taken all this out. I was thinking how this is intended for local developer use - so the whole docker registry thing probably shouldn't exist here now. We took out the docker-push section etc.

Currently it has this 'phony' docker build:

>	docker build -t $(docker_repository):$(version) .

to build locally (which makes sense) but then this run...

>	docker run -i -t -p $(exposed_port):$(exposed_port) --env-file=local_env $(docker_registry)/$(docker_repository):$(version)

So, for consistency, I'll just remove all reference to the ECR docker_registry from the makefile.